csvcubed/csvcubed/utils/qb/cube.py

```diff
@@ -151,9 +164,73 @@ def _validate_observation_value_constraints(cube: Cube) -> List[ValidationError]
             errors += _validate_observation_value(obs_val_column, multi_units_columns)
             errors += _validate_single_measure_cube(cube, obs_val_column)
 
+        errors += _validate_missing_observation_values(cube, observed_value_columns[0])
+
     return errors
 
 
+def _validate_missing_observation_values(
```

Comment:
I think that the observations should be in their own module.

Reply:
I think at some point we'll end up splitting out all of the different validations into separate files. What makes you suggest it now?

Reply:
It's grown this module by 50% LoC, which is some artibtrary mental threshold for some minor refactoring. Won't hurt if you don't want to do it now, but it is easily done now.